apps/consumer-client/src/pages/examples/add-pcd.tsx

```diff
@@ -570,7 +580,12 @@ async function addGPCPCD(
   podFolder: string | undefined
 ): Promise<void> {
   await GPCPCDPackage.init?.({
-    zkArtifactPath: ZUPASS_URL + "artifacts/test/proto-pod-gpc"
+    zkArtifactPath: gpcArtifactDownloadURL(
+      GPC_ARTIFACT_CONFIG.source as GPCArtifactSource,
```

Comment:
why not define the type `GPC_ARTIFACT_CONFIG`'s field `source` to be `GPCArtifactSource` rather than `string`, etc.?

Reply:
Because it's parsed from JSON without being checked (which seemed duplicative), and I didn't want to force a dependency in @pcd/client-shared on @pcd/gpc just for a type.